k8s/gitlab/staging/release.yaml

```diff
@@ -70,9 +70,70 @@ data:
       value: 3
     - op: remove
       path: /spec/values/gitlab/gitlab-shell/service
+    - op: remove
+      path: /spec/valuesFrom/0/secretKeyRef
+    - op: remove
+      path: /spec/values/global/psql/port
+    - op: remove
+      path: /spec/values/global/psql/database
+    - op: remove
+      path: /spec/values/global/psql/username
     - op: replace
-      path: /spec/valuesFrom/0/secretKeyRef/name
-      value: gitlab-{ENV}-secrets
-    - op: replace
-      path: /spec/values/global/psql/password/secret
-      value: gitlab-{ENV}-secrets
+      path: /spec/values/global/psql/password
+      value: {}
+    - op: add
+      path: /spec/values/postgresql
+      value:
```

Comment:
Not sure how to know whether there's anything missing here other than try it out.  Couple questions: Is there a limit on the size of the values for these patch entries?  Did you try this one out to see if it applies cleanly?

Reply:
Yup, I ran it through the `gitops_patch.py` script and it outputted what is expected.